src/viam/rpc/server.py

```diff
@@ -114,3 +131,21 @@ async def create_and_serve(
         """
         server = cls(components)
         await server.serve(host, port, log_level, path=path)
+
+
+def wrapper(func: Callable):
```

Comment:
would add a test or two for this

Reply:
Should this be in a new `test_server` file or is there somewhere we can fit this?